pex/bin/pex.py

```diff
@@ -458,10 +458,26 @@ def configure_clp_pex_environment(parser):
         default=[],
         type="str",
         action="append",
-        help="The Python interpreter to use to build the pex.  Either specify an explicit "
-        "path to an interpreter, or specify a binary accessible on $PATH. This option "
-        "can be passed multiple times to create a multi-interpreter compatible pex. "
-        "Default: Use current interpreter.",
+        help=(
```

Comment:
Reworded. I wanted to add a little insight into how Pex only sometimes searches for interpreters.